crates/uv/tests/run.rs

```diff
@@ -1362,9 +1362,9 @@ fn run_from_directory() -> Result<()> {
     3.12.[X]
 
     ----- stderr -----
-    warning: `VIRTUAL_ENV=[VENV]/` does not match the project environment path `.venv` and will be ignored
+    warning: `VIRTUAL_ENV=.venv` does not match the project environment path `project/.venv` and will be ignored
     Using Python 3.12.[X] interpreter at: [PYTHON-3.12]
-    Creating virtual environment at: .venv
+    Creating virtual environment at: project/.venv
```

Comment:
I think this is correct... because the project environment _is_ at `project/.venv` relative to the current working directory.